pages/profile.tsx

```diff
@@ -1,37 +1,63 @@
 import React from "react";
-import { Container, ProgressBar } from "react-bootstrap";
+import { Button, Container, ProgressBar, Image } from "react-bootstrap";
+import Link from "next/link";
+import { FaExpand } from "react-icons/fa";
+import { useUser } from "../lib/hooks/auth";
 
 export default function Profile() {
+  const { user } = useUser();
   return (
-    <Container className="mt-5 text-center d-grid gap-2">
-      <h1>Statistics</h1>
-      <table className="Statistics">
-        <thead>
-          <tr>
-            <th scope="col">12</th>
-            <th scope="col">92</th>
-            <th scope="col">6</th>
-            <th scope="col">6</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td scope="row">Played</td>
-            <td>Win %</td>
-            <td>Current Streak</td>
-            <td>Maximum Streak</td>
-          </tr>
-        </tbody>
-      </table>
-      <Container className="mt-5 text-center d-grid gap-2">
-        <h1> Guess Distribution </h1>
-        <ProgressBar now={100 / 4} label={`1`} variant="success" />
-        <ProgressBar now={0} label={`0`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 1} label={`4`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-        <ProgressBar now={100 / 4} label={`1`} variant="secondary" />
-      </Container>
-    </Container>
+    <div className="container">
+      <div className="row">
+        <div className="col-sm-6 text-center">
+          <br />
+          <br />
+          <br />
+          <br />
+          <Image
+            src={user?.profilePicture}
+            alt="Profile picture"
+            width="200"
+            height="200"
+            roundedCircle
+          />
+          <br />
+          <br />
+          <h4>
+            {user?.givenName} {user?.familyName}
+          </h4>
+          <br />
+          <br />
+          <Link href="/game">
+            <Button variant="secondary" size="lg" className="">
+              Real Word of the Day <FaExpand />
+            </Button>
+          </Link>{" "}
+        </div>
+        <div className="col-sm-6">
+          <Container className="mt-5">
+            <Link href="/profile">
+              <Button variant="secondary" size="sm" className="">
+                Dashboard
+              </Button>
+            </Link>{" "}
+            <Link href="/statistics">
+              <Button variant="light" size="sm">
+                Statistics
+              </Button>
+            </Link>
+          </Container>
+          <Container className="mt-5 d-grid gap-2"></Container>
+          <h1> Board </h1>
+          <h1 className="text-centered">
+            ⬜⬜⬜⬜⬜ <br />
+            ⬜⬜⬜⬜⬜ <br />
+            ⬜⬜⬜⬜⬜ <br />
+            ⬜⬜⬜⬜⬜ <br />
+            ⬜⬜⬜⬜⬜ <br />
```

Comment:
You can get the user's board from the user profile that is returned from the backend in `useUser`, so you should be able to display the completed board here and replace this placeholder. Take a look at `pastGuesses` in `IUser`.